vmdb/app/models/pg_lock.rb

```diff
@@ -0,0 +1,29 @@
+class PgLock < ActiveRecord::Base
```

Comment:
It's up to you.  I named it this because it matches the table name, so we don't have to set the table name manually.